### ISSUE TITLE
Big reactors Balance issue

### ISSUE THREAD

**Issue by ImperaServ** [covers1624/PowerConverters#29]
The steam producer allow you to produce more power using the steam than the power used to create it. 
The biggest thing I saw with this is a self-powered turbine from Big Reactors, generating 28kRF/t for free and using around 1.8kRF/t to generate its steam.
[http://i.imgur.com/AXFI56b.png](url)

**Comment by covers1624** [comment]
Yeah this is a known issue, I have spent may hours trying to find a way to balance this but sadly none are really practical..

**Comment by florian96** [comment]
It is not a exploit from the Power Converters. It is a Extploit from Big Reactors. But you can reduce the RF Generation rate in the BigReactors Config.
